ionic2/app/components/poi-card/poi-card.component.html

```diff
@@ -1,46 +1,59 @@
-<div class="slide-card" #slideCard @slide="slideState">
-  <div class="poke-poi-card-content" *ngIf="pokePOI" [ngClass]="'poi-type-'+pokePOI.getType()">
-    <ion-row>
+<div id="slide-card" #slideCard @slide="slideState">
+  <!-- Sightings -->
+  <div *ngIf="poi && poi.type == 'sighting' && pokemon">
```

Comment:
you could make use of the Elvis operator here: `<div *ngIf="poi?.type == 'sighting' && pokemon">`